day-2-definitions/rabbitmq/templates/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
```

Comment:
Why do you need the ingress?

Reply:
To have access to the RabbitMQ admin console. Another option would be to access through port-forward

Reply:
But why? That is a highly debug oriented UI, why would you want it exposed?

Reply:
I did following the same approach than for other services like ceph, Prometheus stack, safe, etc